semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolInformationOps.scala

```diff
@@ -215,8 +215,13 @@ trait SymbolInformationOps { self: Scalacp =>
     }
 
     private def annotations: List[s.Annotation] = {
-      // FIXME: https://github.com/scalameta/scalameta/issues/1315
-      Nil
+      sym match {
+        case c: ScalaSigSymbol =>
+          c.attributes.map(attribute => s.Annotation(attribute.typeRef.toSemanticTpe))(
+            collection.breakOut)
```

Comment:
Let's replace this use of `breakOut` with `.toList`. breakOut is an advanced feature, so I'd be wary of using it when a simple alternative exists.